src/console_cmd.c

```diff
@@ -1210,6 +1209,7 @@ TbBool cmd_exec(PlayerNumber plyr_idx, char *msg)
                         }
                     }
                 }
+                //todo replace with any on digger_list
```

Comment:
Did you forget this, or is this for a future PR?

Reply:
it's just for cheats, so yeah could be tackled at some point but really not critical at all